src/components/Calculator.js

```diff
@@ -21,41 +21,47 @@ class Calculator extends Component {
           </div>
+          <div className="calculator">
+            <div className="display">
+              { total}
+              {operation}
+              {next}
 
-          <div className="numbers">
-            <button onClick={this.clickEvent} type="button" color="primary" name="AC">AC</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="">+/-</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="">%</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="2">2</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="3">3</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="4">4</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="5">5</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="6">6</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="7">7</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="8">8</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="9">9</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name="0">0</button>
-            <button onClick={this.clickEvent} type="button" color="primary" name=".">.</button>
+            </div>
+            <div className="operators">
+              <button onClick={this.clickEvent} name="÷" type="button" color="primary" value="">&#247;</button>
+              <button onClick={this.clickEvent} name="x" type="button" color="primary" value="">&times;</button>
+              <button onClick={this.clickEvent} name="-" type="button" color="primary" value="">-</button>
+              <button onClick={this.clickEvent} name="+" type="button" color="primary" value="">+</button>
+              <button onClick={this.clickEvent} name="=" type="button" color="primary" value="">=</button>
+            </div>
+
+            <div className="numbers">
+              <button onClick={this.clickEvent} type="button" color="primary" name="AC">AC</button>
+              <button onClick={this.clickEvent} type="button" color="primary" name="">+/-</button>
+              <button onClick={this.clickEvent} type="button" color="primary" name="">%</button>
+              <button onClick={this.clickEvent} type="button" color="primary" name="2">2</button>
```

Comment:
- [ ] (OPTIONAL) Kindly fix the functionalities of the `+/-` and `%` buttons as right now they do not seem to be working. This will improve the UX of your site.
- [ ] (OPTIONAL) Also, when a user clicks on the all-clear button, `AC` it would be great if you return a `0` rather than removing all numbers on the input 👍